.scalafmt.conf

```diff
@@ -1,24 +1,20 @@
-version = 2.1.0
+version = 2.7.5
```

Comment:
Should we go all the way to 3.0.3?

Reply:
ok, I tried that, see https://github.com/lightbend/akkaserverless-java-sdk/pull/364/commits/f41a9ef1fecb9f9341bc6444773a8de585a95692 and https://github.com/lightbend/akkaserverless-java-sdk/pull/364/commits/0c6df07e9fd6337573fa512393465ec5861a0caa